src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.ts

```diff
@@ -253,4 +251,10 @@ export class DotEditLayoutGridComponent implements OnInit, ControlValueAccessor
     private isHaveRows(): boolean {
         return !!(this.value && this.value.rows && this.value.rows.length);
     }
+
+    private resizeGrid(timeOut?): void {
+        setTimeout(() => {
+            this.ngGrid.triggerResize();
+        }, timeOut ? timeOut : 0);
```

Comment:
Are you sure we want ```0``` here? https://stackoverflow.com/questions/779379/why-is-settimeoutfn-0-sometimes-useful

Reply:
yes. there are times when we don't want to wait more than the time effect that creates the timeout